src/app/external-cluster-wizard/steps/external-cluster/provider/eks/component.ts

```diff
@@ -268,8 +298,8 @@ export class EKSClusterSettingsComponent
           roleArn: this.controlValue(Controls.RoleArn),
           version: version,
           vpcConfigRequest: {
-            subnetIds: this.controlValue(Controls.SubnetIds),
-            securityGroupIds: this.controlValue(Controls.SecurityGroupsIds),
+            subnetIds: this.controlValue(Controls.SubnetIds)[ComboboxControls.Select],
+            securityGroupIds: this.controlValue(Controls.SecurityGroupsIds)[ComboboxControls.Select],
```

Comment:
```suggestion
            subnetIds: this.controlValue(Controls.SubnetIds)?.[ComboboxControls.Select],
            securityGroupIds: this.controlValue(Controls.SecurityGroupsIds)?.[ComboboxControls.Select],
```

Please also update Line# 322